### PR TITLE
Update NexusMods.com.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

**Comment by koops76** [comment]
@J0WI Need MCB testing, if OK, merge.

File: src/chrome/content/rules/nexusmods.com.xml [hunk 1]
```diff
@@ -1,13 +1,52 @@
...
+	<target host="static-3.nexusmods.com" />
+	<target host="static-4.nexusmods.com" />
+	<target host="static-5.nexusmods.com" />
+	<target host="static-6.nexusmods.com" />
+	<target host="static-7.nexusmods.com" />
+	<target host="static-8.nexusmods.com" />
+	<target host="staticdelivery.nexusmods.com" />
+	<target host="tes.nexusmods.com" />
+	<target host="wfto.nexusmods.com" />
+	<target host="wiki.nexusmods.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/nexusmods.com.xml]: Still major MCB issues

**Comment by J0WI** [comment]
What about #11961?

File: src/chrome/content/rules/nexusmods.com.xml (hunk 1 above)
Comment (koops76) [reply src/chrome/content/rules/nexusmods.com.xml]: Can you tell me which targets have MCB issues?

**Comment by koops76** [comment]
@J0WI Closed in favor of this PR.

File: src/chrome/content/rules/nexusmods.com.xml (hunk 1 above)
Comment (J0WI) [reply src/chrome/content/rules/nexusmods.com.xml]: Only this one `wiki.nexusmods.com` (see removed comment on top)

**Comment by koops76** [comment]
@J0WI Done.

**Pull request merged**